prow/scripts/cluster-integration/kyma-integration-k3d-app-connector-components-os.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
It looks like these two yamls, I mean this one and the `...skr.yaml` one are duplicates of [these two files](https://github.com/kyma-project/kyma/tree/main/tests/components/application-connector/resources/installation-config). If it's possible I'd suggest using only one pair of these files to avoid desynchronization problems